frontend/src/Component/Chat/Controls.purs

```diff
@@ -125,8 +128,13 @@ handleAction = case _ of
   SendMessage ev → do
     liftEffect $ Event.preventDefault ev
     H.modify_ _ { message { inputValue = "" } }
-  KeyPressed k →
-    if shiftKey k then pass
-    else case key k of
-      "Return" → pure unit -- needs last version of `web-events`
+  KeyPressed keyEv → do
+    if shiftKey keyEv then pass
```

Comment:
This doesn't look like its required as @ line 140 it will `pass` anyway

Reply:
I don't get what you mean - if we don't check `Shift` then it will submit our form whether it is pressed or not, and in the line 140 we have to pass as we need to cover all the variants